backend/controller/lmevaljob_controller.go

```diff
@@ -826,3 +809,24 @@ func argsToString(args []lmevalservicev1beta1.Arg) string {
 	}
 	return strings.Join(equalForms, ",")
 }
+
+func generateEnvs(secrets []lmevalservicev1beta1.EnvSecret) []corev1.EnvVar {
+	var envs = []corev1.EnvVar{}
+	for _, secret := range secrets {
```

Comment:
This looks like where we'd add an extra conditional to pull the value from a file

Reply:
mounting the secret as files is different from the ENV, so I handle them separately. check `patch4FileSecrets` function in `lmevaljob_controller.go`